exchange/howto/requirements-for-adding-an-altcoin.adoc

```diff
@@ -0,0 +1,66 @@
+* At each new release we will check if already added coins/tokens have reached the min. usage requirements (see below). If those requirements are not met the coin/token will be removed. Adding the token again requires a statement about the changed circumstances (e.g. link to discussions where users demand for the coin/token is documented,...).
+* For requesting again a coin/token which got rejected by voting it requires a clear statement about the changed circumstances. Objecting the political positions is not sufficient (e.g. Coin XXX is not a scamcoin because of ....).
+* PR which fail to fulfill the formal requirements (e.g. failing address validation unit tests) will result that the coin/token will be blocked for 3 months.
+
+[*] The Bisq DAO will be executed in a manual manner as long the full DAO version is not implemented.
+See: link to DAO phase 0 doc. (TODO)
+
+
+== Formal requirements
+
+* The PR need to use the below form and has answered all questions.
+* There need to be an active block explorer available.
+* The ticker symbol must not conflict with one of the national currency ISO 4217 ticker symbols listed at: https://en.wikipedia.org/wiki/ISO_4217
+* The ticker symbol must not conflict with one of the top 100 crypto-currency ticker symbols at time when the PR gets created listed at: https://coinmarketcap.com/currencies/
+* The PR need to contain code for address validation [2] of the coin/token and unit tests[3] for that validation.
+* The PR need to contain code which adds the coin to the list of coins/tokens [4]. Use the optional "true" parameter if the coin is a token (no dedicated blockchain). Add the coin according to alphabetic order based on the name (not ticker).
+* The requirements to add the coin does not cause extra effort for Bisq like adding additional input fields (e.g. if the address alone is not sufficient for the trade).
+* The arbitrator need to be able to see the transaction in the block explorer by using the receiver address. In case that is not possible as a result of enhanced privacy features (Zcash, Monero, CryptoNote coins,...) a suggested solution need to be added to the PR (e.g. in Zcash we support only transparent addresses not z-addresses; in Monero we require that the user is using a Monero wallet which supports transaction keys to enable 3rd party audits of the transaction which is used in case of a dispute). As such additional cased require additional effort (popup with background info) it will be up to Bisq contributors to add a PR for the popup and translation strings. Of course the code or the popup can be part of the PR as well but a more detailed description is out of scope of that document.
+
+
+== Form to be filled out and be pasted into the PR
+
+* Ticker symbol (e.g. LTC): ...
+* Official coin name (e.g. Litecoin): ...
+* Official block explorer URL: ...
+* Is it an Altcoin (dedicated blockchain) or a token (anything else) (e.g. Altcoin): ...
+* Official coin/token project URL: ...
+
```

Comment:
You can indent the whole form so that it shows up in a fixed-width font complete with markup. Note that the markup format should be Markdown, not Asciidoc, as that's what people will be pasting into the PR.